src/IconButton/README.md

```diff
@@ -47,20 +54,21 @@ notes: ''
 
 ```jsx live
 () => {
-  const variants = ["primary", "secondary", "success", "warning", "danger", "light", "dark"]
+  const icon = FontAwesome.faBars;
+  const variants = ["brand", "primary", "secondary", "success", "warning", "danger", "light", "dark"];
 
   return (
     <div className="d-flex">
       {variants.map((variant) => {
         return (
-          <div class={`bg-${variant} p-2 m-2`}>
+          <div class={`bg-${variant} p-1`}>
```

Comment:
@raq929 re: square inverse variant, they're not square there's just a background behind them because the accent colors are white. They'd disappear otherwise

Reply:
oooooh, got it.